.github/workflows/release.yaml

```diff
@@ -3,16 +3,48 @@ on:
   push:
     tags:
       - v*
-
+env:
+  REGISTRY: ghcr.io
+  IMAGE_NAME: ${{ github.repository }}
 jobs:
   dist:
     runs-on: ubuntu-latest
+    needs:
+      - lint
+      - test
+      - build
+      - e2e
+      - docker
```

Comment:
Are you sure you can reference jobs that are in a different workflow? Besides, every job except maybe build can be skipped. I don't see a reason to test again upon Git tag pushes

Reply:
That makes sense. I removed the `need` attribute.